common/changes/@itwin/property-grid-react/persist-show-hide-null-per-user_2022-07-22-16-49.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Persist show/hide null value toggle per user and not per iTwin/iModel",
+      "type": "patch"
```

Comment:
the is a major version bump change since its a breaking change

Reply:
I didn't have the option for major change. It was minor or patch.

Reply:
I can go back and do minor if you want

Reply:
okay it's minor now, everybody calm down 😜